lib/docx/body-reader.js

```diff
@@ -312,8 +312,10 @@ function BodyReader(options) {
             
             var gridSpan = properties.firstOrEmpty("w:gridSpan").attributes["w:val"];
             var colSpan = gridSpan ? parseInt(gridSpan, 10) : 1;
+            var isTableHeader = properties.children && properties.children[0].attributes['w:firstColumn'] === '1';
             
             var cell = documents.TableCell(children, {colSpan: colSpan});
+            cell.isTableHeader = isTableHeader;
```

Comment:
Rather than mutating the object, passing in `isTableHeader` in the same way as `colSpan` would be preferred.